index.html

```diff
@@ -1288,29 +1303,41 @@ <h4>
   <section> <h3>Push a smart poster message</h3>
     <pre class="example">
       const writer = new NDEFWriter();
+      const encoder = new TextEncoder();
       writer.push({ records: [
         {
-          recordType: "smart-poster",
+          recordType: "smart-poster",  // Sp
           data: { records: [
             {
-              recordType: "url",
+              recordType: "url",  // URL record for the Sp content
               data: "https://my.org/content/19911"
```

Comment:
Should this be something like `"https://example.com/funny-dance.gif"` so that reader can easily match this with the `"t"` type record?

Reply:
Let's do this in a next PR. We need to add creating SP anyway.

Reply:
As long as it's in the a follow-up PR, I'm good.